docs/tutorials/getting_started.md

```diff
@@ -0,0 +1,520 @@
+
+- Name: `Tutorial`
+- MCC: `001`
+- MNC: `01`
+- UPF: `upf-external.private5g.svc.cluster.local:8805`
+- gNodeB: `private5g-cu-cu (tac:1)`
+
+You should see the following network slice created:
+
+```{image} ../images/nms_network_slice.png
+:alt: NMS Network Slice
+:align: center
+```
+
+Navigate to Subscribers and create a new subscriber with the following attributes:
+
+- IMSI: `001010100007487`
+- OPC: `981d464c7c52eb6e5036234984ad0bcf`
+- Key: `5122250214c33e723a5dd523fc145fc0`
+- Sequence Number: `16f3b3f70fc2`
+- Network Slice: `Tutorial`
+- Device Group: `Tutorial-default`
+
+You should see the following subscriber created:
+
+```{image} ../images/nms_subscriber.png
+:alt: NMS Subscriber
+:align: center
+```
+
+```{note}
+Due to current limitations in the network configuration procedure, it is required to restart the CU Pod after configuring the network.
+This limitation will be addressed in the future.
+To restart the CU Pod execute:
+`microk8s.kubectl -n private5g delete pod cu-0`
+```
+
+## 4. Deploy Charmed OAI RAN DU
+
+Add Charmed OAI RAN DU Terraform module to `main.tf`:
+
+```console
+cat << EOF >> main.tf
+module "du" {
+  source = "git::https://github.com/canonical/oai-ran-du-k8s-operator//terraform"
+
+  model_name = juju_model.private5g.name
+  config     = {
+    "simulation-mode": true
+  }
+}
+
+resource "juju_integration" "du-cu" {
+  model = juju_model.private5g.name
+  application {
+    name     = module.du.app_name
+    endpoint = module.du.fiveg_f1_endpoint
+  }
+  application {
+    name     = module.cu.app_name
+    endpoint = module.cu.fiveg_f1_endpoint
+  }
+}
+
+EOF
+```
+
+Update Juju Terraform provider:
+
+```console
+terraform init
+```
+
+Deploy the gNB simulator:
+
+```console
+terraform apply -auto-approve
+```
+
+Monitor the status of the deployment:
+
+```console
+juju status --watch 1s --relations
+```
+
+The deployment is ready when the `du` application is in the `active/idle` state.
+
+## 5. Deploy Charmed OAI RAN UE Simulator
+
+Add Charmed OAI RAN UE Terraform module to `main.tf`:
+
+```console
+cat << EOF >> main.tf
+module "ue" {
+  source = "git::https://github.com/canonical/oai-ran-ue-k8s-operator//terraform"
+
+  model_name = juju_model.private5g.name
+}
+
+resource "juju_integration" "ue-du" {
+  model = juju_model.private5g.name
+  application {
+    name     = module.ue.app_name
+    endpoint = module.ue.fiveg_rfsim_endpoint
+  }
+  application {
+    name     = module.du.app_name
+    endpoint = module.du.fiveg_rfsim_endpoint
+  }
+}
+
+EOF
+```
+
+Update Juju Terraform provider:
+
+```console
+terraform init
+```
+
+Deploy the gNB simulator:
+
+```console
+terraform apply -auto-approve
+```
+
+Monitor the status of the deployment:
+
+```console
+juju status --watch 1s --relations
+```
+
+The deployment is ready when the `ue` application is in the `active/idle` state.
+
+## 6. Run 5G network traffic simulation
+
+Run the simulation:
+
+```console
+juju run ue/leader start-simulation
```

Comment:
The simulation does not succeed. UE startup commands in DU charm logs seems different the commands which is provided in UE charm.

Reply:
This is because the UE was not updated after changing the DU configuration.
[Here's a PR fixing the problem](https://github.com/canonical/oai-ran-ue-k8s-operator/pull/20)